eslint.config.mjs

```diff
@@ -0,0 +1,272 @@
+import stylistic from "@stylistic/eslint-plugin";
+import globals from "globals";
+import js from "@eslint/js";
+
+export default [
```

Comment:
Automatically converted from .eslintignore, .eslintrc.yml via https://eslint.org/docs/latest/use/migrate-to-9.0.0#-new-default-config-format-eslintconfigjs + https://eslint.org/docs/latest/use/configure/migration-guide